dist/src/main/java/io/camunda/webapps/controllers/IndexController.java

```diff
@@ -29,7 +29,7 @@ public String index() {
    * Redirects the old frontend routes (common tasklist and operate routes) to the default-app
    * sub-path. This can be removed after the creation of the auto-discovery service.
    */
-  @GetMapping({"/processes", "/login"})
+  @GetMapping({"/processes"})
```

Comment:
❓ Are we now stopping the redirect of `/login` for any specific reason? I was trying to think of if this impacts us with the Identity login page